CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Change Log
+
+## v1.19.0 - 11/25/2021
+
+[1.20.0 Milestone](https://github.com/eclipse-theia/theia/milestone/26)
+
+<a name="breaking_changes_1.20.0">[Breaking Changes:](#breaking_changes_1.20.0)</a>
+
 - [plugindev] Renamed HostedPlugin(Server|Client) to PluginDev(Server|Client)
+- [core] `value` is now not optional in `Deferred<T>.resolve(value: T)`
+- [core] `T` defaults to `void` if not specified when defining a `Deferred`
```

Comment:
I don't _think_ this should count as a breaking change, since previously you _had_ to specify the type in a generic, and now you don't if you mean `void`, but no one's existing generics should be invalidated by that change?

Reply:
I noticed a few places where `T` was omitted but TS default used to be `any` in that case, now it is `unknown`, and only by having `void` can we call `.resolve()` without a value... So I thought it would be worth mentioning that we default to void now.

Reply:
Never hurts to provide more information. After I mentioned it, I was working in some Deferred-heavy areas and noticed the same thing. 👍